internal/rbd/rbd_util.go

```diff
@@ -1703,6 +1703,16 @@ func cleanupRBDImageMetadataStash(metaDataPath string) error {
 	return nil
 }
 
+// expand checks if the requestedVolume size and the existing image size both
+// are same. If they are same, it returns nil else it resizes the image.
+func (rv *rbdVolume) expand() error {
```

Comment:
I was expecting to see `expand()` to have a `size` parameter... I guess this works too.

Reply:
instead of passing one more input using struct fields internally

Reply:
instead of passing one more input using struct fields internally